ui/app/components/buy-button-subview.js

```diff
@@ -83,17 +106,92 @@ BuyButtonSubview.prototype.render = function () {
           account: props.account,
         }),
       ]),
-      h('h3.text-transform-uppercase', {
+
+      h('.flex-row', {
         style: {
-          paddingLeft: '15px',
-          fontFamily: 'Montserrat Light',
-          width: '100vw',
-          background: 'rgb(235, 235, 235)',
-          color: 'rgb(174, 174, 174)',
-          paddingTop: '4px',
-          paddingBottom: '4px',
+          alignItems: 'center',
+          justifyContent: 'center',
+        },
+      }, [
+        h('h3.text-transform-uppercase.flex-center', {
+          style: {
+            paddingLeft: '15px',
+            width: '100vw',
+            background: 'rgb(235, 235, 235)',
+            color: 'rgb(174, 174, 174)',
+            paddingTop: '4px',
+            paddingBottom: '4px',
+          },
+        }, 'Select Service'),
+      ]),
+
+    ])
+
+  )
+}
+
+
+BuyButtonSubview.prototype.primarySubview = function () {
+  const props = this.props
+  const network = props.network
+
+  switch (network) {
+    case 'loading':
+      return
+
+    case '1':
+      return this.mainnetSubview()
+
+    // Ropsten, Rinkeby, Kovan
+    case '3':
+    case '4':
+    case '42':
+    const networkName = networkNames[network]
```

Comment:
could L149-L174 be indented?  looks like it's all meant to be under `case '42'` -- maybe either the ternary on L165 isn't needed, since we only run that code when the network is 42